index.html

```diff
@@ -360,6 +360,79 @@ <h6 class='card-horizontal-title'>CARD TITLE</h6>
     </div>
   </div>
 
+  <h2>Image design</h2>
+      <img
+        src="https://picsum.photos/id/212/200/300"
+        alt="image"
+        class="image image-round"
+      />
+
+      <img
+        src="https://picsum.photos/id/212/200/300"
+        alt="image"
+        class="image image-square"
+      />
+
+      <h4>Responsive Image design</h4>
+      <div class="image-responsive-container">
+        <img
+          src="https://dragonball.guru/wp-content/uploads/2021/01/goku-dragon-ball-guru.jpg"
+          alt="image"
+          class="image-responsive"
+        />
+      </div>
+
+      <h2>Input Components</h2>
+      <h4>Form Inputs</h4>
+    
+    
+          <form class='form'>
+            <div class='form-group'>
+              <label><b>EMAIL</b></label>
+              <input type='email' placeholder="Enter your email"/>
+            </div>
+            <div class='form-group'>
+              <label><b>PASSWORD</b></label>
+            <input type='password' placeholder="Enter your password"/>
+          </div>
+          <div>
+            <button class='btn btn-icon btn-secondary'>
+              <i class="fas fa-comment-alt "></i>
+              sign in
+            </button>
+          </div>
+          </form>
+    
+          <form class='form'>
+              <textarea class="textbox" placeholder='Enter Your Message' ></textarea>
+            
+            <button class='btn btn-icon btn-secondary'>
+              <i class="fas fa-comment-alt "></i>
+              sign in
+            </button>
+          </div>
+          </form>
+    
+    
+          <h4>Error Form Inputs </h4>
+          <form class='form'>
+            <span class="msg-error">Enter correct Email and Password</span>
+            <div class='form-group-error'>
+              <label><b>EMAIL</b></label>
+              <input  type='email' placeholder="Enter your email"/>
+            </div>
+            <div class='form-group-error'>
```

Comment:
You can try to use the same form-group class here. As the code is common in both the classes